src/csvcubed/models/cube/qb/components/measure.py

```diff
@@ -47,7 +63,23 @@ def get_permitted_rdf_fragment_hints(self) -> Set[RdfSerialisationHint]:
     def get_default_node_serialisation_hint(self) -> RdfSerialisationHint:
         return RdfSerialisationHint.Component
 
-    _measure_uri_validator = validate_uri("measure_uri")
+    _measure_uri_validator = pydantic_validate_uri("measure_uri")
+
+    def validate_data(
```

Comment:
Get rid of this function please and sort out the imports.

Reply:
Removed it and cleaned up imports.